includes/admin/feedzy-rss-feeds-usage.php

```diff
@@ -213,6 +217,65 @@ public function get_usage_stats() {
 		return $stats;
 	}
 
+	/**
+	 * Record the import count per year week.
+	 * 
+	 * @return void
+	 * 
+	 * @since 5.0.8
+	 */
+	public function record_import_per_week() {
+		$data_time  = current_datetime();
+		$curr_week  = intval( $data_time->format( 'W' ) );
+		$curr_year  = intval( $data_time->format( 'Y' ) );
+		$curr_month = intval( $data_time->format( 'n' ) );
```

Comment:
I think here records might be malformed in some edge cases because of the formatting. We should make sure that `W` and `Y` are 100% correlated. IF the standards don't match, you might have offset weeks. E.g. for 30 Dec 2025, the week might be recorded as the first week of 2026. What would that mean is that you record the year correctly (2025, but it would use week 1 and back-record data).